packages/clarity-js/src/interaction/pointer.ts

```diff
@@ -57,9 +57,10 @@ function touch(event: Event, root: Node, evt: TouchEvent): void {
             let y = "clientY" in entry ? Math.round(entry["clientY"] + d.scrollTop) : null;
             x = x && frame ? x + Math.round(frame.offsetLeft) : x;
             y = y && frame ? y + Math.round(frame.offsetTop) : y;
+            const id = "identifier" in entry ? entry["identifier"] : undefined;
```

Comment:
The server side is assuming id=0 is the primary pointer and the rest are != o. However there is no guarantee in teh spec of that behavior. We need to do the number ourselves and map from the arbitrary touch id to the simple numbering scheme.

Basic idea: 

* Keep a dictionary of id -> touch number
* When it's a touchstart event then add a new entry to the dictionary with the touch # assigned starting from 0.

Reply:
The `identifier` is not arbitrary, it's 0-based and it resets when all the fingers are off the screen. It's not a part of the spec, but that's the implemented behaviour in browsers (tested on Chrome-based ones as well as Firefox).

Reply:
Can you please add a comment that capture the above (starts at zero and confirmed on the 3 major browse engines?

Reply:
done